From 7e555748e2ad77da55d84d7be59a0c527f72ea2f Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Mon, 16 Jan 2017 17:41:35 +0100
Subject: No auto update when upgrading

---
 azurelinuxagent/common/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/azurelinuxagent/common/conf.py b/azurelinuxagent/common/conf.py
index 7911699..c45ec96 100644
--- a/azurelinuxagent/common/conf.py
+++ b/azurelinuxagent/common/conf.py
@@ -219,7 +219,7 @@ def get_autoupdate_gafamily(conf=__conf__):
 
 
 def get_autoupdate_enabled(conf=__conf__):
-    return conf.get_switch("AutoUpdate.Enabled", True)
+    return conf.get_switch("AutoUpdate.Enabled", False)
 
 
 def get_autoupdate_frequency(conf=__conf__):
-- 
2.9.3

